references/detection/train_tpu.py

```diff
@@ -65,7 +78,15 @@ def get_transform(train):
 def parse_flags():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '--num_cores', type=int, default=None,
+        '--mode', type=str, default='test',
+        choices=('eval', 'test', 'train'),
+        help='eval computes the AP scores, test runs inference on a few sample images, '
+        'train trains the model')
+    parser.add_argument(
+        '--num-eval', type=int, default=50,
+        help='Number of validation set images to use in eval mode')
+    parser.add_argument(
+        '--num_cores', type=int, default=8,
         help='Number of TPU cores to train on (8 for single v2/v3 Cloud TPU)')
```

Comment:
does this work on v2s?

Reply:
The `--num_cores` flag is from @jysohn23's work back in June so I would assume so?

Reply:
presumably w/ small backbones it may work, although it may be valuable to verify that first unless @jysohn23 already verified?

Reply:
IIRC, it should work if batch sizes are small enough.